protocol/v2/ssv/runner/proposer.go

```diff
@@ -266,10 +271,9 @@ func (r *ProposerRunner) ProcessPostConsensus(ctx context.Context, logger *zap.L
 		start := time.Now()
 
 		logger = logger.With(
-			fields.BeaconDataTime(r.metrics.GetBeaconDataTime()),
```

Comment:
why was BeaconDataTime removed? i realize maybe because its tracked in a different way now in metrics, but this doesnt have to affect the log

Reply:
As we discussed - `GetBeaconDataTime` was never recorded, hence it was returning 0